src/utils/htmlTagHelpers.js

```diff
@@ -36,9 +37,22 @@ export function createFactbox(props, children) {
   );
 }
 
-export function createFileList(props) {
-  const id = process.env.NODE_ENV === 'unittest' ? 'testid' : uuid();
-  return render(<FileList {...props} id={id} />);
+export function createFileSection(files, pdfs, heading) {
+  const filelistId = process.env.NODE_ENV === 'unittest' ? 'testid' : uuid();
+  const figureId = process.env.NODE_ENV === 'unittest' ? 'testid' : uuid();
+  return render(
+    <>
+      {files.length && (
+        <FileList files={files} heading={heading} id={filelistId} />
+      )}
+      {pdfs.map((pdf, index) => (
+        <Figure key={`${index}-${figureId}`} id={`${index}-${figureId}`}>
+          <h2>{`${pdf.title} (PDF)`}</h2>
+          <iframe title={pdf.title} height="600" src={pdf.formats[0].url} />
```

Comment:
Er den høyden piksler? Og hvordan vil den høyden være om man bruker `article-converter` i `ndla-frontend` også, der sikkert mange brukere er på mobil? 
Kan være jeg har jeg misforstått 😅

Reply:
Også usikker på kva som er lur standardbredde her. Har du forresten vurdert å bruke makeIframe fra pluginHelpers for å definere iframen? 
Trur også ikkje du treng (PDF) i h2-taggen.

Reply:
> Er den høyden piksler? Og hvordan vil den høyden være om man bruker `article-converter` i `ndla-frontend` også, der sikkert mange brukere er på mobil?
Høyden er i piksler. Det var forøvrig et godt spørsmål med ndla-frontend, som også tror må oppdateres. Det ser ikke ut som den bruker samme endepunkt for article-converter til å vise fram artikler. Prøvde å kjøre opp ndla-frontend lokalt med converter og den viser ikke pdf i iframe.

> Har du forresten vurdert å bruke makeIframe fra pluginHelpers for å definere iframen?
Denne ser ut til å returnere iframe som en streng, så det vil vel ikke fungere med mindre det parses tilbake til html.

Reply:
Husk at du må kjøre både graphql og article-converter lokalt om du skal teste det med ndla-frontend-

Reply:
Glem det angående ndla-frontend. Det ser ut til å fungere bra. Tok bare litt tid før ny publisert versjon av artikkel ble tilgjengelig.